### PR TITLE
Move to GitHub merge queues.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

**Comment by vext01** [comment]
Any squashing to be done?

**Comment by ltratt** [comment]
I think the two commits should be separate.

**Comment by jacob-hughes** [comment]
@vext01 This one should be ready to re-review and merge.

**Comment by vext01** [comment]
I think @ltratt is better qualified for yksom changes.

**Comment by jacob-hughes** [comment]
Sure, but I think you'll need to do the final sign off because Laurie opened the PR.

**Comment by ltratt** [comment]
Yes, @vext01 you need to review this please.

File: src/lib/vm/objects/block.rs [hunk 1]
```diff
@@ -79,6 +79,8 @@ pub struct Block {
 // this explicit impl is needed to tell the compiler we are not doing that.
 unsafe impl FinalizerSafe for Block {}
 
+unsafe impl Sync for Block {}
```
Comment (vext01) [inline src/lib/vm/objects/block.rs:82]: Do you need a comment saying why this is safe?

Bearing in mind I don't know much about the GC :)

Comment (jacob-hughes) [reply src/lib/vm/objects/block.rs:85]: Sure! I've documented it here: e43d30559ed564cd3e9720c04bb74af9b4793c88

Comment (vext01) [reply src/lib/vm/objects/block.rs:85]: Sorry, I meant for the various `unsafe impl Sync for Blah {}`s.

Comment (ltratt) [reply src/lib/vm/objects/block.rs:85]: AFAICS all of them are now documented.

**Comment by vext01** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Comment by vext01** [comment]
sorry, i didn't see it had failed.

**Comment by jacob-hughes** [comment]
It's a rustfmt problem. I've fixed it here: 9aa838a6591e8824616d1685670a36d68071cc52, let me know if I can squash.

**Comment by jacob-hughes** [comment]
Squashed

**Comment by vext01** [comment]
Did you forget to sign the commit?

**Comment by jacob-hughes** [comment]
Weird! My gpg-agent has failed. Can I force-push an update?

**Comment by vext01** [comment]
Please push a signed commit.

**Comment by jacob-hughes** [comment]
Force-pushed a signed commit.

**Comment by jacob-hughes** [comment]
Ugh I’ve not seen that one before.

**Comment by vext01** [comment]
```
16:41:47 thread 'main' panicked at src/lib/vm/core.rs:404:13:
16:41:47 Not enough stack space to execute block.
```

ouch.

**Comment by vext01** [comment]
Any news on this one?

**Comment by ltratt** [comment]
That's one for @jacob-hughes.

**Comment by jacob-hughes** [comment]
@vext01 You should be able to merge this one now :)

**Pull request merged**